core/web/resolver/vrf_test.go

```diff
@@ -0,0 +1,301 @@
+package resolver
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/smartcontractkit/chainlink/core/services/keystore"
+	"github.com/smartcontractkit/chainlink/core/services/keystore/keys/vrfkey"
+)
+
+func TestResolver_GetVRFKey(t *testing.T) {
+	t.Parallel()
+
+	query := `
+		query GetVRFKey($id: ID!) {
+			vrfKey(id: $id) {
+				... on VRFKeySuccess {
+					key {
+						id
+						compressed
+						uncompressed
+						hash
+					}
+				}
+
+				... on NotFoundError {
+					message
+					code
+				}
+			}
+		}
+	`
+
+	fakeKey, err := vrfkey.NewV2()
```

Comment:
I think there's a `cltest.DefaultVRFKey` you can use, which speeds up the tests bc we don't have to repeat all the keygen crypto